contracts/core/extensions/CoreIssuance.sol

```diff
@@ -226,18 +215,29 @@ contract CoreIssuance is
     )
         internal
     {
+        // Verify Set was created by Core and is enabled
+        require(
```

Comment:
The only material change in this PR. Moved checks into internal function whereas before, these checks weren't happening in `CoreIssuanceOrder`.